Sources/AuthFoundation/Token Management/Internal/KeychainTokenStorage.swift

```diff
@@ -15,71 +15,47 @@
 import Foundation
 
 class KeychainTokenStorage: TokenStorage {
-    static let serviceName = "com.okta.authfoundation.keychainStorage"
-    static let defaultTokenName = "com.okta.authfoundation.keychainStorage.default"
+    static let serviceName = "com.okta.authfoundation.keychain.storage"
+    static let metadataName = "com.okta.authfoundation.keychain.metadata"
+    static let defaultTokenName = "com.okta.authfoundation.keychain.default"
 
     weak var delegate: TokenStorageDelegate?
     
-    private lazy var _defaultTokenId: String? = {
-        if let defaultResult = try? Keychain
-            .Search(account: KeychainTokenStorage.defaultTokenName)
-            .get()
-        {
-            return String(data: defaultResult.value, encoding: .utf8)
+    private(set) lazy var defaultTokenID: UUID? = {
```

Comment:
Might be simpler/more robust to make  it a `String`? UUID is fine of course but can't think of many benefits and you need to keep converting back and forth given that keychain can't store this type.